cherab/jet/machine/cad_files.py

```diff
@@ -672,10 +672,22 @@
            VACUUM_VESSEL + DIAGNOSTICS + IL_SC + IL_SC_STRUCTURE + DIVERTOR_TILES + DIVERTOR_STRUCTURE
 
 
-def import_jet_mesh(world, override_material=None, tungsten_material=None, beryllium_material=None,
+def import_jet_mesh(world, mesh_description=JET_MESH, override_material=None, tungsten_material=None, beryllium_material=None,
```

Comment:
Please add this new argument to the end of the list. Otherwise existing codes which call this function with positional rather than keyword arguments wlil break.

Reply:
Good point, I shifted it to the end.